xsl/webhelp-common.xsl

```diff
@@ -256,7 +256,7 @@ border: none; background: none; font-weight: none; color: none; }
 
 	.ui-layout-pane { /* all 'panes' */
 		background: #FFF;
-		border: 1px solid #BBB;
+		*/border: 1px solid #BBB;*/
```

Comment:
Remove it if you don't want it. git can remember it for us, we don't need to carry around commented CSS